ui-react/src/components/topBar/ReportProblemModal.tsx

```diff
@@ -0,0 +1,244 @@
+import React, { useState } from "react";
+import { Button, Form, Modal, Row, Col, Stack } from "react-bootstrap";
+import { openExternal } from "../../eventsBridge";
+import { repository } from "../../../package.json";
+import os from "os";
+// import LogsCanvas from "./LogsCanvas";
+import { isValidEmail } from "../../utils/validations";
+import { getZIndexes } from "../../utils/zIndexesManager";
+
+type ReportProblemForm = {
+  title?: string;
+  email?: string;
+  details?: string;
+};
+
+type ReportProblemModalProps = {
+  show: boolean;
+  onClose: () => void;
+};
+
+function ReportProblemModal({ show, onClose }: ReportProblemModalProps) {
+  const [form, setForm] = useState<ReportProblemForm>({
+    title: "",
+    email: "",
+    details: "",
+  });
+
+  const [errors, setErrors] = useState({});
+  // const [showLogs, setShowLogs] = useState(false);
+
+  const setField = (field: string, value: string) => {
+    setForm((prevForm) => ({ ...prevForm, [field]: value }));
+
+    if (!!errors[field]) setErrors({ ...errors, [field]: null });
+  };
+
+  const validateForm = (validateEmail = true) => {
+    const newErrors = {};
+
+    if (!form.title || form.title === "") newErrors.title = "שדה חובה";
+
+    if (validateEmail) {
+      if (!form.email || !isValidEmail(form.email))
+        newErrors.email = "שדה מייל לא חוקי";
+    }
+
+    return newErrors;
+  };
+
+  const openGithub = (e) => {
+    e.preventDefault();
+
+    const formErrors = validateForm(false);
+    if (Object.keys(formErrors).length > 0) {
+      setErrors(formErrors);
+      return;
+    }
+
+    const url = createGithubIssueLink(
+      form.title,
+      form.details,
+      form.attachedLogs ?? ""
+    );
+    console.info(`Open bug report url with title: ${form.title}`);
+    openExternal(url);
+  };
+
+  //TODO: SOURCE_COMMIT_SHORT should be taken from env file
+  const createGithubIssueLink = (title: string, details: string, log: any) => {
+    const formattedDetails = details
+      ? `
+        ## Details
+        
+        ${details}`
+      : "";
+
+    const formattedLog = log
+      ? `
+        ## Log
+        \`\`\`
+        ${log}
+        \`\`\``
+      : "";
+
+    const sysInfo = `
+        ## System Info
+        
+         - Source Version: \`${"SOURCE_COMMIT_SHORT" || "unknown"}\`
+         - OS: \`${os.platform()}${os.arch()}\`
+         - OS Version: \`${os.release()}\`
+        `;
+
+    return `${`${repository}/issues/new?` +
+      `title=${encodeURIComponent(title)}` +
+      "&body="}${encodeURIComponent(
+      formattedDetails + formattedLog + sysInfo
+    )}`;
+  };
+
+  const sendReport = (e) => {
+    e.preventDefault();
+
+    const formErrors = validateForm(true);
+    if (Object.keys(formErrors).length > 0) {
+      setErrors(formErrors);
+      return;
+    }
+
+    // const eventId = Sentry.userReportProblem(
+    //   form.title,
+    //   form.details,
+    //   form.attachedLogs ?? "",
+    //   form.email
```

Comment:
But I think this is needed, it is not related to the *logs* problem.

Sometimes users clicks on it and I'm getting their report on Sentry.

Reply:
Currently, the Sentry.ts file is sitting in the backend just like the logger.ts
and both are using some parameters from the backend env file.
so we should discuss how to use both of them in the new React project, and where the parameters should be held in that case.

But I think it should be in another PR.

Reply:
Yes, Ok, but it is not good that you commented only the Sentry send function, because now the user will press on **Send** and it will do nothing.

Better is to comment our the button or replace it to open the Discord.

Reply:
Sure, I missed this one.
Fixed.